Assets/Scripts/Framework/Gameplay/Interactions/Grabbables/Grabbable.cs

```diff
@@ -115,13 +118,28 @@ public virtual void PlaceOnGround()
 
             transform.position = hit.point + new Vector3(0, BoxCollider.size.y * .5f, 0);
 
-            if(ObjectParentUtils.TryGetObjectParent(hit.collider.gameObject, out ObjectParent objectParent))
+            if (ObjectParentUtils.TryGetObjectParent(hit.collider.gameObject, out ObjectParent objectParent))
             {
                 objectParent.SetParent(NetworkObject);
+                PlayGrabbableSound(true);
                 return;
             }
 
             NetworkObject.UnsetParent();
+            PlayGrabbableSound(true);
+        }
```

Comment:
You can just have 1 `PlayGrabbableSound()` call above the ObjectParent if statement.